core/utils/utils_test.go

```diff
@@ -252,3 +252,56 @@ func TestMinBigs(t *testing.T) {
 		})
 	}
 }
+
+func TestMaxUint(t *testing.T) {
+	tests := []struct {
```

Comment:
I would prefer not to use a table test if there is only one test, maybe add some other cases?